README.md

```diff
@@ -1,3 +1,224 @@
-# mod1
-mod1
-this is added
\ No newline at end of file
+<div>
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
```

Comment:
- [ ] Dear @demesameneshoa, I have seen the previous reviewers have made a suggestion to update your readme file content with related information. As part of that, I have left you some comments to help you address them. Remember, the template is designed by Microverse for us all to use to professionally document our projects and have it filled with content related to each project we may be working on.